### ISSUE TITLE
question about running in parallel

### ISSUE THREAD

**Issue by ajdneuro12** [BIDS-Apps/freesurfer#48]
Hi all,

Thanks for the great app! I am trying to run a large number of participants but I am having trouble running in parallel. I am running it on an hpc with singularity. I have tried specifying multiple participants with the participant label and changing the number of cpus but it is not running in parallel. Thanks for your help! 

Best,
Alex

**Comment by Shotgunosine** [comment]
Hi @ajdneuro12, sorry for the incredibly delayed response. I would suggest running one subject per job on the HPC. Will that work for you? I'm not sure if this is still an issue or not, but in some cases running multiple subjects on the same node along with using openmp parallelization for some steps can result in more CPUs than available being requested.

**Comment by alexlicohen** [comment]
I agree with this, submit one job per subject, then you can go a group level analysis later if needed.

-Alex
